File: ES-CLIENT/engineUpdate.cpp
#include "engineUpdate.h"

EngineUpdate::EngineUpdate() {
	Status = "Connected";
	Name = "";
	cylinderCount = 0;
	RPM = 0;
	maxRPM = 0;
	sparkAdvance = 0;
	tps = 0;
	vehicleSpeed = 0;
	sparkTimingList = {};
	manifoldPressure = 0;
	gear = -1;
	clutchPosition = 0;
	atLimiter = false;
	twoStepActive = false;
	engineLoad = 0;
	power = 0;
	torque = 0;
	airSCFM = 0;
	afr = 0;
	temperature = 0;
}

File: ES-GUI/engineUpdate.cs
﻿using System;
using System.Collections.Generic;


namespace ES_GUI
{
    public class engineUpdate
    {
        public string Status;
        public string Name;
        public int cylinderCount;
        public double RPM;
        public double maxRPM;
        public double sparkAdvance;
        public double tps;
        public double vehicleSpeed;
        public List<double> sparkTimingList;
        public double manifoldPressure;
        public int gear;
        public double clutchPosition;
        public bool atLimiter;
        public bool twoStepActive;
        public double engineLoad;
        public double power;
        public double torque;
        public double airSCFM;
        public double afr;
        public double temperature;
    }
}
